minitest-stub_on_roids.gemspec

```diff
@@ -1,6 +1,6 @@
 Gem::Specification.new do |s|
   s.name        = "minitest-stub_on_roids"
-  s.version     = "0.0.3"
+  s.version     = "0.0.4"
```

Comment:
Remove this please, I'll bump the version in a separate commit before the release.